client/src/App.vue

```diff
@@ -7,6 +7,7 @@
     export default {
         name: 'app'
     };
-    export const API_ENDPOINT = window.location.protocol+"//"+window.location.hostname+":8090"
-    export const WEBSOCKET_ENDPOINT = "wss://"+window.location.hostname+":8091"
+    export const REST_API_ENDPOINT = window.location.protocol+"//"+window.location.hostname
+    export const REST_API_PORT = "8090"
+    export const WEBSOCKET_PORT = "8091"
```

Comment:
Hmm not sure if it is just me, but didn't you completely remove the websocket now or?

Reply:
Oh wait nvm found it :D
It looks good :)